src/Constants/authorization.js

```diff
@@ -0,0 +1,118 @@
+// Spotify application details
```

Comment:
I personaly find it a bit odd that we have functions in the constants folder but maybe I just haven't been exposed to this convention. I might've named it utils for utility functions.

Reply:
Name of folder changed to reflect more accurately what is in the folder.